test/opt/def_use_test.cpp

```diff
@@ -33,6 +33,22 @@ using ::testing::UnorderedElementsAreArray;
 using namespace spvtools;
 using spvtools::opt::analysis::DefUseManager;
 
+uint32_t NumUses(const std::unique_ptr<ir::IRContext> &context, uint32_t id) {
```

Comment:
Add a method comment.

Reply:
LLVM has method hasNUsesOrMore which can sometimes be faster than counting all the uses, but is a quick way to tell if a thing has any uses or a unique use.    In future we might want something similar as an Instruction method.

Reply:
Will do.